src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -196,6 +211,17 @@ internal partial class Session
                     helper, recentItems, filterText, filterResults);
             }
 
+            private Boolean IsAfterDot(Model model, ITextSnapshot textSnapshot, Dictionary<TextSpan, string> textSpanToText)
```

Comment:
Nit: use `bool` instead of `Boolean`.  And change your option, and bug @jasonmalinowski to fix the bug in the option.